### PR TITLE
[IMP] partner_event: Do single browse

### PR DESCRIPTION
**Pull request by yajo**
If adding 1000 partners, we save now 999 queries.

@Tecnativa

### CODE REVIEW

File: partner_event/wizard/res_partner_register_event.py [hunk 1]
```diff
@@ -27,10 +27,9 @@ def _prepare_registration(self, partner):
     @api.multi
     def button_register(self):
         registration_obj = self.env['event.registration']
-        partner_obj = self.env['res.partner']
         errors = []
-        for partner_id in self.env.context.get('active_ids', []):
-            partner = partner_obj.browse(partner_id)
+        for partner in self.env['res.partner'].browse(
+                self.env.context.get('active_ids', [])):
```
Comment (pedrobaeza) [inline partner_event/wizard/res_partner_register_event.py:32]: Incorrect indentation

Comment (yajo) [reply partner_event/wizard/res_partner_register_event.py:32]: It's accepted in PEP8 for `if` blocks, so I guess for `for` blocks too.

Comment (pedrobaeza) [reply partner_event/wizard/res_partner_register_event.py:32]: Ah, sorry, my bad. I though it was the first line inside the block.

**Review by pedrobaeza — APPROVED** [review]
@chienandalu @rafaelbn please review

**Review by sergio-teruel — APPROVED** [review]

**Comment by rafaelbn** [comment]
@Yajo travis is red

**Review by carlosdauden — APPROVED** [review]

**Comment by carlosdauden** [comment]
If you want you can fix unrelated pylint errors :wink:

**Comment by pedrobaeza** [comment]
Pylint arreglado. Mergeando.

**Pull request merged**